### PR TITLE
[254-llm-chatbot]update chatglm3's prompt template

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**
Including:
 1. update the NNCF dependency with it develop branch
 2. since the chatbot responses from current prompt template do not work perfect for chatglm3,  I updated chatglm3's chatbot prompt template with its unique tokenizer methods. 
 https://huggingface.co/THUDM/chatglm3-6b/blob/main/tokenization_chatglm.py#L201

### CODE REVIEW

**Comment by OpenVINO-dev-contest** [comment]
@eaidova pls help to review and merge, thanks a lot.

**Comment by eaidova** [comment]
@OpenVINO-dev-contest thanks for update, please add notebooks into https://github.com/openvinotoolkit/openvino_notebooks/blob/main/.ci/ignore_pip_conflicts.txt

Alos as new release is soon and nncf underactive development for next release features, possibly it is better to use release branch instead of develop:
https://github.com/openvinotoolkit/nncf/tree/release_v280

File: notebooks/254-llm-chatbot/254-llm-chatbot.ipynb [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "%pip uninstall -q -y openvino-dev openvino openvino-nightly\n",
     "%pip install -q --extra-index-url https://download.pytorch.org/whl/cpu\\\n",
     "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
```
Comment (eaidova) [inline notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:60]: there are some issues with new commits usage in optimum-intel for llms, I prepared fix, but all responsible for merge people are on holiday, so I'll add workaround, but in separate PR

Comment (eaidova) [reply notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:60]: https://github.com/openvinotoolkit/openvino_notebooks/pull/1575

Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:60]: shall I update it in this PR ?

Comment (eaidova) [reply notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:60]: no, I just let you know, it is global issue that should be fixed in many notebooks, so it is better to keep it separately

**Comment by OpenVINO-dev-contest** [comment]
> @OpenVINO-dev-contest thanks for update, please add notebooks into https://github.com/openvinotoolkit/openvino_notebooks/blob/main/.ci/ignore_pip_conflicts.txt
> 
> Alos as new release is soon and nncf under active development for next release features, it is better to use the release branch instead of develop: https://github.com/openvinotoolkit/nncf/tree/release_v280

done

**Review by eaidova — APPROVED** [review]

File: .ci/ignore_pip_conflicts.txt [hunk 2]
```diff
@@ -6,6 +6,8 @@ notebooks/212-pyannote-speaker-diarization/212-pyannote-speaker-diarization.ipyn
 notebooks/230-yolov8-optimization/230-yolov8-instance-segmentation.ipynb  # ultralytics==8.0.43
 notebooks/230-yolov8-optimization/230-yolov8-keypoint-detection.ipynb  # ultralytics==8.0.159
 notebooks/230-yolov8-optimization/230-yolov8-object-detection.ipynb  # ultralytics==8.0.43
+notebooks/254-llm-chatbot/254-llm-chatbot.ipynb # nncf@https://github.com/openvinotoolkit/nncf/tree/release_v280
+notebooks/254-llm-chatbot/254-rag-chatbot.ipynb# nncf@https://github.com/openvinotoolkit/nncf/tree/release_v280
```
Comment (eaidova) [inline .ci/ignore_pip_conflicts.txt:10]:
```suggestion
notebooks/254-llm-chatbot/254-rag-chatbot.ipynb # nncf@https://github.com/openvinotoolkit/nncf/tree/release_v280
```

**Pull request merged**